ts/backend/src/app/features/utils/aggregator.mts

```diff
@@ -0,0 +1,500 @@
+// TODO: move interfaces to common and merge with common interfaces
+
+/*
+REMARK:
+
+Definition tree is like this:
+
+BenchmarkGroupDefinition - a group of benchmark definitions
+└ BenchmarkDefinition[] - benchmark, group of test definitions
+  └ TestDefinition[] - available tests
+    └ ScenarioDefinition[] - defined scenarios per test
+
+
+Aggregated/scored structure is like this:
+
+BenchmarkGroupScored - a group of benchmark definitions
```

Comment:
This level does neve exist afaics. Rounds of a competition run in parallel and validation campaigns run in parallel as technically independent benchmarks.